rhino/src/main/java/org/mozilla/javascript/Scriptable.java

```diff
@@ -33,10 +33,10 @@ public interface Scriptable {
      *
      * <p>See ECMA 8.6.2 and 15.2.4.2.
      */
-    public String getClassName();
+    String getClassName();
```

Comment:
What the reasoning for removing `public` from all the interface method?

Reply:
From the Java Spec: https://docs.oracle.com/javase/specs/jls/se11/html/jls-9.html#jls-9.4

> A method in the body of an interface may be declared public or private ([§6.6](https://docs.oracle.com/javase/specs/jls/se11/html/jls-6.html#jls-6.6)). If no access modifier is given, the method is implicitly public. It is permitted, but discouraged as a matter of style, to redundantly specify the public modifier for a method declaration in an interface.

`SymbolScriptable` (a newer interface) already omitted the `public` modifiers, but `Scriptable` did not. I was adding additional methods to `Scriptable` and wanted to keep them consistent.